tracing-subscriber/src/lib.rs

```diff
@@ -77,6 +77,10 @@
     unused_parens,
     while_true
 )]
+
+#![allow(clippy::cognitive_complexity)]
+#![allow(clippy::match_wild_err_arm)]
```

Comment:
Is this something where we can change the code in a minor way rather than requiring the allow attribute? In general, I'd prefer not to put global allow attributes on whole crates unless it's absolutely necessary.

Reply:
I wasn't able to easily see how to fix the `cognitive_complexity` lint but maybe. It should be possible to make smaller changes for the other one. I'll check.